CONTRIBUTING.md

```diff
@@ -1,93 +1,23 @@
-## Getting Started
+This document serves as a reference for the Mirador community and others interested in the project. It aims to outline agreed upon practices that are used in the development and maintenance of the Mirador software.
 
-Mirador uses [node.js](http://nodejs.org/) and a build system to assemble, test, and manage the development resources. If you have never used these tools before, you may need to install them.
+## Contributing to Mirador
+Contributions to Mirador are always welcome, however, it will always be helpful to begin any large change by submitting an issue or engaging with the Mirador community. For more on this, see [plugin architecture](#plugin-architecture). Mirador 3.0 and beyond adheres to [semantic versioning](https://semver.org/) providing adopters and contributors to better rely on what changes can be expected in released versions of the software.
 
- 1. Install Node, if you haven't already (available at the link above)  
- 1. Clone the mirador repository (if you haven't already done so above); `git clone https://github.com/ProjectMirador/mirador.git`
- 1. On the command line, go into the mirador folder
- 1. Install all dependencies with `npm install`. Run `npm start`
+All contributions should be submitted as a [GitHub pull request](https://help.github.com/articles/about-pull-requests/) to the `master` branch. Pull requests must be reviewed and accepted by another Mirador maintainer and pass all continuous integration checks. Contributions should have tests for the feature or bug fix, documentation and should conform to the Mirador agreed upon coding style. Contributions should not include a “built” version of Mirador, this will help in reducing merge conflicts.
 
-`npm start` will run a local server that is available at `localhost:8000`.
+### Adding dependencies
+Careful consideration should be given when adding software dependencies to Mirador. Considerations should be given to: added size of the dependency, whether or not that dependency is maintained and tested, how the dependency may interact with Mirador embedded in other environments.
```

Comment:
```suggestion
Careful consideration should be given when adding software dependencies to Mirador. During the code review process, new dependencies may be evaluated on the following considerations: 
- added size of the dependency
- whether or not that dependency is maintained and tested
- how the dependency may interact with Mirador embedded in other environments
```

Reply:
This is just a small stylistic suggestions to emphasize that each of these aspects are important.